### ISSUE TITLE
[SCREENREADER]: New messages must be announced consistently across assistive technology software

### ISSUE THREAD

**Issue by ianpsant** [microsoft/BotFramework-WebChat#3936]
# Please view our [Technical Support Guide](https://github.com/microsoft/BotFramework-WebChat/tree/main/docs/TECHNICAL_SUPPORT_GUIDE.md) before filing a new issue.

## Version

<meta name=\"botframework-directlinespeech:version\" content=\"4.12.0\">
<meta name=\"botframework-webchat:bundle:variant\" content=\"full-es5\">
<meta name=\"botframework-webchat:bundle:version\" content=\"4.12.0\">
<meta name=\"botframework-webchat:core:version\" content=\"4.12.0\">
<meta name=\"botframework-webchat:ui:version\" content=\"4.12.0\">

## Describe the bug

As a screen reader user, I need to know when new content is posted to the chat log. This includes my questions, and the bot's responses.

## Steps to reproduce

Two screen reader combinations I tested weren't announcing new content completely:

IE11 + JAWS didn't announce my question or the chatbot's responses
Firefox + NVDA didn't announce my questions, but did announce the chatbot's responses
Live regions need to announce information properly. This is a key part of the experience for screen reader users; they won't know new content is available unless their AT starts reading back the new content. Focus remains on the input field, and may be problematic when the bot is offering links to action for veterans. Maybe consider a focus management state that sets focus on the user's question, then moves focus to the bot's answer when it arrives. Or if a focus management solution is not possible, see if the live regions can be announced consistently.

## Expected behavior

New content is announced to all screen readers, consistently

## Additional context

https://github.com/department-of-veterans-affairs/va.gov-team/issues/25168

[Bug]

**Comment by compulim** [comment]
This should be resolved. I verified that the live region should announce both user's question and bot's answer consistently in live region. It should work according to https://github.com/microsoft/BotFramework-WebChat/blob/main/docs/ACCESSIBILITY.md#ux-narration-of-messages.
